src/ert/config/surface_config.py

```diff
@@ -18,16 +22,72 @@
 class SurfaceConfig(ParameterConfig):  # pylint: disable=too-many-instance-attributes
     ncol: int
     nrow: int
-    xori: int
-    yori: int
-    xinc: int
-    yinc: int
-    rotation: int
+    xori: float
+    yori: float
+    xinc: float
+    yinc: float
+    rotation: float
     yflip: int
     forward_init_file: str
     output_file: Path
     base_surface_path: str
 
+    @classmethod
+    def from_config_list(cls, surface: List[str]) -> Self:
```

Comment:
Does `surface` always have context? In that case it is a `ContextList[str]`

Reply:
The context for the surface is always the `List[str]`. Which is actually `ContextList[FileContextToken]`